.github/workflows/black-fix.yml

```diff
@@ -0,0 +1,37 @@
+name: black
+
+on: [push, pull_request]
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  black:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        ref: ${{ github.head_ref }}
+
+    - name: Install black
+      run: |
+        sudo apt update -y
+        sudo apt -y install python3-pip python3-setuptools python3-wheel
+        pip3 install black
+
+    - name: Version
+      run: |
+        python3 --version
+        $HOME/.local/bin/black --version
+
+    - name: Run black
+      run: |
+        pwd
+        ls
+        $HOME/.local/bin/black tests/ tools/ $(grep '^#.*python.*$' bin/ tests/ tools/ -Ir|cut -d: -f1|grep -v _boutcore_build)
+
+    - uses: stefanzweifel/git-auto-commit-action@v4
+      with:
+        commit_message: Apply black changes
```

Comment:
Does this trigger another CI run? If so, might be good to put `[skip CI]` in the message

Reply:
Yes it does, I added [skip CI] :+1: